src/python/pants/base/build_environment.py

```diff
@@ -49,7 +53,7 @@ def get_pants_configdir():
   return os.path.expanduser(os.path.join(config_home, 'pants'))
 
 
-def get_default_pants_config_file():
+def get_default_pants_config_file(*, buildroot_sentinel_filename: str = "pants") -> str:
```

Comment:
Unused argument?

Reply:
Oops, not intentionally unused! I didn’t finish the change. Thanks!